Descent3/init.cpp

```diff
@@ -1571,7 +1571,7 @@ void InitIOSystems(bool editor) {
     strcpy(Base_directory, GameArgs[0]);
     int len = strlen(Base_directory);
     for (int i = (len - 1); i >= 0; i--) {
-      if (i == '\\') {
+      if (Base_directory[i] == '\\' || Base_directory[i] == '/') {
         Base_directory[i] = '\0';
```

Comment:
you need to exit the loop using `break;` here when the first path separator is found.